samples/reuse-downloaded-chrome/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Linq;
+using PuppeteerSharp;
+
+namespace Example.ReuseDownloadedChrome
+{
+    class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            Console.WriteLine("This example downloads the default version of Chromium to a custom location");
+
+            var currentDirectory = Directory.GetCurrentDirectory();
+            var downloadPath = Path.Combine(currentDirectory, "..", "..", "CustomChromium");
+            Console.WriteLine($"Attemping to set up puppeteer to use Chromium found under directory {downloadPath} ");
+
+            if (!Directory.Exists(downloadPath))
+            {
+                Console.WriteLine("Custom directory not found. Creating directory");
+                Directory.CreateDirectory(downloadPath);
+            }
+
+            Console.WriteLine("Downloading Chromium");
+
+            var browserFetcherOptions = new BrowserFetcherOptions { Path = downloadPath };
+            var browserFetcher = new BrowserFetcher(browserFetcherOptions);
+            await browserFetcher.DownloadAsync(BrowserFetcher.DefaultRevision);
+
+            var executablePath = CreateExecutablePath(downloadPath);

```

Comment:
`BrowserFetcher` has a [GetExecutablePath](https://github.com/kblok/puppeteer-sharp/blob/master/lib/PuppeteerSharp/BrowserFetcher.cs#L206) method that will do that for you.

Reply:
Sorry I should have checked the API. Thanks for the tip.